src/llmq/quorums_instantsend.h

```diff
@@ -52,23 +52,100 @@ class CInstantSendDb
 public:
     CInstantSendDb(CDBWrapper& _db) : db(_db) {}
 
+    /**
+     * This method is called when an InstantSend Lock is processed and adds the lock to the database instantsend.dat
```

Comment:
"instantsend.dat" is mentioned multiple times...but this file is unused now actually. "instantsend.dat" was a plain dump based "DB", while we now use leveldb (the `llmq` db/folder).